Makefile

```diff
@@ -12,18 +12,19 @@ SOURCES = packages codemods
 build: clean
 	make clean-lib
 	./node_modules/.bin/gulp build
-ifneq ("$(BABEL_ENV)", "cov")
-	make build-standalone
```

Comment:
If we leave this out locally, I think our unit tests will fail because it'll try to run the tests for standalone.

Reply:
I mentioned this in that but it looks like we only run tests in circle ci

Reply:
The standalone tests specifically don't run for CI because of https://github.com/babel/babel/blob/master/scripts/_get-test-directories.sh#L14, but they definitely run when doing local development, so if we skip building this for local dev, they tests will fail.

Reply:
I think we could explicitly include babel-standalone when wanted, instead enabling it by default and excluding it from cov.